Python/compile.c

```diff
@@ -71,11 +71,24 @@
         ((C)->c_flags.cf_flags & PyCF_ALLOW_TOP_LEVEL_AWAIT) \
         && ((C)->u->u_ste->ste_type == ModuleBlock))
 
+struct compiler;
+
+typedef _PyInstruction instruction;
+typedef _PyInstructionSequence instr_sequence;
+
+static instr_sequence *compiler_instr_sequence(struct compiler *c);
+static int compiler_future_features(struct compiler *c);
+static struct symtable *compiler_symtable(struct compiler *c);
+static PySTEntryObject *compiler_symtable_entry(struct compiler *c);
+
+#define INSTR_SEQUENCE(C) compiler_instr_sequence(C)
+#define FUTURE_FEATURES(C) compiler_future_features(C)
+#define SYMTABLE(C) compiler_symtable(C)
+#define SYMTABLE_ENTRY(C) compiler_symtable_entry(C)
```

Comment:
How much do the macros buy us? The actual function names are simple enough and only slightly longer. It might be simpler to just drop the indirection.

Reply:
After we split the compiler to a separate file, and start thinking about making things configurable, the indirection might turn out useful (you can redefine the macro to do something else). I don't know if we will, but wanted to keep the option for now.

Reply:
I'd be inclined to add it later if it's needed (yagni and all), but I don't feel strongly about it. Whatever works for you :-).